src/NHibernate/Util/ReflectHelper.cs

```diff
@@ -565,7 +651,7 @@ internal static object GetConstantValue(string qualifiedName, ISessionFactoryImp
 			return null;
 		}
 
-		[Obsolete("Please use Linq.ReflectionHelper instead")]
+		[Obsolete("Please use Linq.ReflectHelper instead")]
```

Comment:
Well, search&replace does not make it for this obsolete reason. I would fix it tomorrow for triggering another build by the way.